scripts/lmdk/build_module.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
```

Comment:
Since there are more than one module that can be build I will call this `build_modules.py`.